diff --git a/app-editors/atom/ChangeLog b/app-editors/atom/ChangeLog
index 18cc836..0b18d13 100644
--- a/app-editors/atom/ChangeLog
+++ b/app-editors/atom/ChangeLog
@@ -2,6 +2,9 @@
 # Copyright 1999-2014 Gentoo Foundation; Distributed under the GPL v3
 # $Header: $
 
+  21 Apr 2018; shimataro <shimataro@zelkova.cc> +atom-1.26.0.ebuild:
+  Version bump.
+
   07 Apr 2018; shimataro <shimataro@zelkova.cc> +atom-1.25.1.ebuild:
   Version bump.
 
diff --git a/app-editors/atom/Manifest b/app-editors/atom/Manifest
index f6656a6..d36682c 100644
--- a/app-editors/atom/Manifest
+++ b/app-editors/atom/Manifest
@@ -13,6 +13,7 @@ DIST atom-1.24.0-amd64.rpm 129891855 BLAKE2B d1d12978bc2ecdc65ae4233bceaacc5cc06
 DIST atom-1.24.1-amd64.rpm 130121924 BLAKE2B 143db676e8a31f2e90177fc3f97af5889061384ad88c952c2bdbe15fcf3672b88f8e48cb75113a2b4d472eaefc9686a41752c1515501866a1bfa412395bcb707 SHA512 155086bfb7ec4459c05d57de73021126192ac34cd4906b4ea9de2ffce1afe1c4d7c6ccce06f78fff43db1011b062f93e47c1304a6cb0e39374a5e4e20f363dad
 DIST atom-1.25.0-amd64.rpm 139577962 BLAKE2B 798d38456dfdeae6fde65ce9f415745f09145a2f68ca55b8108631316b55f72410e5e3ea2cfdecac874e0aab85e0eea43a153e17e82543681858c5f2c975aea5 SHA512 d20b88a764fe865f2f8770f26089aa5c3649b1068b5f9b7e33f3c2cf52868a679fa8f2089747bde8546d78fb7ba4c00799db25f9220bbe843b159b4e28d1fa6c
 DIST atom-1.25.1-amd64.rpm 139568721 BLAKE2B dfb755e6173a0fb32a612fa23544fe9446e43f08a56b536779bbac29dba82689c30adfb10d781d5c90e1eb051a5a16ca238e1241f705322bb74411ab7e440ab7 SHA512 af14668da2b187b5a6836f0eb4ad25f45bdfcbef5759d8944330ca585b8f886c4928a6fb220acc6befeffbc5ad00cf28f985efd3a1fda47cc33750d5bdacc287
+DIST atom-1.26.0-amd64.rpm 140489769 BLAKE2B 39cce5b45679e5e3143824e9749aa2e2f1f2a3df8fd056df27da97852456f707c61964089e7c9ff769355baf075ef3098e2cc7b340e00409f0b3f392052046fa SHA512 fbe0d45682b898428732944eb1041cef5b98402c2649d386b18db73eb5700ed50632d4387d4777b39c657719bef8ea67195967c9c9254a6e18a09e847ce01dfc
 EBUILD atom-1.20.0.ebuild 672 BLAKE2B c45523f5394c684d2f27431407eeeb1e421d7a94cbe1a9c5f46757cdd7e465dd273c682e4110434b43b9d2c13a2e5f8e39182ef9b530fde7228eabfddc755cb5 SHA512 eed467e567d2cade85ec6d129a991a080ac90a98fa1e0a55f69fafdb9f9694b8c3bb6861fdff63dfc56283602662d4bbdb85319c55a42919084b206fef11bbb8
 EBUILD atom-1.20.1.ebuild 672 BLAKE2B c45523f5394c684d2f27431407eeeb1e421d7a94cbe1a9c5f46757cdd7e465dd273c682e4110434b43b9d2c13a2e5f8e39182ef9b530fde7228eabfddc755cb5 SHA512 eed467e567d2cade85ec6d129a991a080ac90a98fa1e0a55f69fafdb9f9694b8c3bb6861fdff63dfc56283602662d4bbdb85319c55a42919084b206fef11bbb8
 EBUILD atom-1.21.0.ebuild 672 BLAKE2B c45523f5394c684d2f27431407eeeb1e421d7a94cbe1a9c5f46757cdd7e465dd273c682e4110434b43b9d2c13a2e5f8e39182ef9b530fde7228eabfddc755cb5 SHA512 eed467e567d2cade85ec6d129a991a080ac90a98fa1e0a55f69fafdb9f9694b8c3bb6861fdff63dfc56283602662d4bbdb85319c55a42919084b206fef11bbb8
@@ -28,5 +29,6 @@ EBUILD atom-1.24.0.ebuild 672 BLAKE2B c45523f5394c684d2f27431407eeeb1e421d7a94cb
 EBUILD atom-1.24.1.ebuild 672 BLAKE2B c45523f5394c684d2f27431407eeeb1e421d7a94cbe1a9c5f46757cdd7e465dd273c682e4110434b43b9d2c13a2e5f8e39182ef9b530fde7228eabfddc755cb5 SHA512 eed467e567d2cade85ec6d129a991a080ac90a98fa1e0a55f69fafdb9f9694b8c3bb6861fdff63dfc56283602662d4bbdb85319c55a42919084b206fef11bbb8
 EBUILD atom-1.25.0.ebuild 672 BLAKE2B c45523f5394c684d2f27431407eeeb1e421d7a94cbe1a9c5f46757cdd7e465dd273c682e4110434b43b9d2c13a2e5f8e39182ef9b530fde7228eabfddc755cb5 SHA512 eed467e567d2cade85ec6d129a991a080ac90a98fa1e0a55f69fafdb9f9694b8c3bb6861fdff63dfc56283602662d4bbdb85319c55a42919084b206fef11bbb8
 EBUILD atom-1.25.1.ebuild 672 BLAKE2B c45523f5394c684d2f27431407eeeb1e421d7a94cbe1a9c5f46757cdd7e465dd273c682e4110434b43b9d2c13a2e5f8e39182ef9b530fde7228eabfddc755cb5 SHA512 eed467e567d2cade85ec6d129a991a080ac90a98fa1e0a55f69fafdb9f9694b8c3bb6861fdff63dfc56283602662d4bbdb85319c55a42919084b206fef11bbb8
-MISC ChangeLog 1018 BLAKE2B 8c88e9e22016e9985836396452a338836613694a0146dae0430aa7255b061bcc135994f12c795e4d98035d3908dcbac0ae0a1ccf92c1e9b460abafca4a5bee1a SHA512 55c3a0c7a6e9ea75e6f758c6bc91b9b0acf9d31574820c3ac259c6428fa7ffe3c1451aeb0858b5bdbc76395a779c47fde595c4459cd4d13179f7a090f6c8c826
+EBUILD atom-1.26.0.ebuild 672 BLAKE2B c45523f5394c684d2f27431407eeeb1e421d7a94cbe1a9c5f46757cdd7e465dd273c682e4110434b43b9d2c13a2e5f8e39182ef9b530fde7228eabfddc755cb5 SHA512 eed467e567d2cade85ec6d129a991a080ac90a98fa1e0a55f69fafdb9f9694b8c3bb6861fdff63dfc56283602662d4bbdb85319c55a42919084b206fef11bbb8
+MISC ChangeLog 1104 BLAKE2B c4eac06aa8149b90a27c08fab9ec960539b5d8e86c7989f0d7da2f870320656f2438700ef2fdfbe1ae2045d2a4b21f82db9cdb7dd0b8619960fd2f89136e598e SHA512 a56f86d449833c92ce30f1e44bea3ef4406a3e63842e9449bcffafc92df9f05a70bdc557bff85492144de9ad705c7d0748d36993c3aa4c2bdd1ffbf4da9e4928
 MISC metadata.xml 398 BLAKE2B 7909b6cea3f0323e54079a5c8bff1353553872e6d70da94dfe97db05e50e817f67405447425ec5aa459f3e87525ff4b39b691db1b24ba9d8697ec6311c4686d7 SHA512 5323b3814b4a843362f7c0f9638f8f7d7707019112d0e4a09248fa8fafb0e84524c92dabd51a354720ab619690e2f73df767e136f1ccbe73ae06290e19d5f2cc
diff --git a/app-editors/atom/atom-1.26.0.ebuild b/app-editors/atom/atom-1.26.0.ebuild
new file mode 100644
index 0000000..e706d75
--- /dev/null
+++ b/app-editors/atom/atom-1.26.0.ebuild
@@ -0,0 +1,40 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=4
+
+inherit rpm xdg-utils
+
+DESCRIPTION="A hackable text editor for the 21st Century"
+HOMEPAGE="https://atom.io/"
+SRC_URI="
+	amd64? ( https://github.com/atom/atom/releases/download/v${PV}/${PN}.x86_64.rpm -> ${P}-amd64.rpm )
+"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="amd64"
+IUSE=""
+
+RDEPEND="
+	gnome-base/gconf
+	gnome-base/libgnome-keyring
+"
+
+S=${WORKDIR}
+
+src_install () {
+	doins -r *
+
+	# change permissions
+	find . -type f -perm -a=x -exec fperms a+x "/{}" \;
+}
+
+pkg_postinst() {
+	xdg_desktop_database_update
+}
+
+pkg_postrm() {
+	xdg_desktop_database_update
+}
