--- a/configure	2020-09-28 20:14:06.564454766 +0900
+++ b/configure	2020-09-28 20:35:52.335758371 +0900
@@ -14651,16 +14651,6 @@
 fi
 
 
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for gtester2xunit dependencies" >&5
-$as_echo_n "checking for gtester2xunit dependencies... " >&6; }
-if !($PYTHON -c "import libxslt, libxml2" 2> /dev/null); then
-  { $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
-$as_echo "no" >&6; }
-  as_fn_error $? "You need to install python-libxslt1 and python-libxml2" "$LINENO" 5;
-fi
-{ $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
-$as_echo "yes" >&6; }
-
 ###########################
 # Headless tests
 ###########################
